docs/ECLLanguageReference/ECLR_mods/BltInFunc-CRON.xml

```diff
@@ -3,104 +3,105 @@
-          </row>
+          <tbody>
+            <row>
+              <entry><emphasis>minute</emphasis></entry>
 
-          <row>
-            <entry><emphasis>hour</emphasis></entry>
+              <entry>An integer value for the minute of the hour. Valid values
+              are from 0 to 59.</entry>
+            </row>
 
-            <entry>An integer value for the hour. Valid values are from 0 to
-            23 (using the 24 hour clock).</entry>
-          </row>
+            <row>
+              <entry><emphasis>hour</emphasis></entry>
 
-          <row>
-            <entry><emphasis>dom</emphasis></entry>
+              <entry>An integer value for the hour. Valid values are from 0 to
+              23 (using the 24 hour clock).</entry>
+            </row>
 
-            <entry>An integer value for the day of the month. Valid values are
-            from 0 to 31.</entry>
-          </row>
+            <row>
+              <entry><emphasis>dom</emphasis></entry>
 
-          <row>
-            <entry><emphasis>month</emphasis></entry>
+              <entry>An integer value for the day of the month. Valid values
+              are from 1 to 31.</entry>
+            </row>
 
-            <entry>An integer value for the month. Valid values are from 0 to
-            12.</entry>
-          </row>
+            <row>
+              <entry><emphasis>month</emphasis></entry>
 
-          <row>
-            <entry><emphasis>dow</emphasis></entry>
+              <entry>An integer value for the month. Valid values are from 1
+              to 12.</entry>
+            </row>
 
-            <entry>An integer value for the day of the week. Valid values are
-            from 0 to 7 (where both 0 and 7 represent Sunday).</entry>
-          </row>
-        </tbody>
-      </tgroup>
-    </informaltable></para>
+            <row>
+              <entry><emphasis>dow</emphasis></entry>
 
-  <para>Any <emphasis>time</emphasis> component that you do not want to pass
-  is replaced by an asterisk (*). You may define ranges of times using a dash
-  (-), lists using a comma (,), and ‘once every n’ using a slash (/). For
-  example, 6-18/3 in the hour field will fire the timer every three hours
-  between 6am and 6pm, and 0-6/3,18-23/3 will fire the timer every three hours
-  between 6pm and 6am.</para>
+              <entry>An integer value for the day of the week. Valid values
+              are from 0 to 7 (where both 0 and 7 represent Sunday).</entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </informaltable></para>
 
-  <para>Example:</para>
+    <para>Any <emphasis>time</emphasis> component that you do not want to pass
+    is replaced by an asterisk (*). You may define ranges of times using a
+    dash (-), lists using a comma (,), and ‘once every n’ using a slash (/).
+    For example, 6-18/3 in the hour field will fire the timer every three
+    hours between 6am and 6pm, and 0-6/3,18-23/3 will fire the timer every
```

Comment:
>and 0-6/3,18-23/3 

is that slightly misleading/confusing ? Since 18-23/3 is only triggered once and could be 18-21/3
I think : 
```
18-21/3,0-6/3
```

might be a little clearer.